estate/models/estate_property.py

```diff
@@ -0,0 +1,50 @@
+from odoo import fields, models
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Properties involved in estate"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
```

Comment:
```suggestion
    name = fields.Char(string="Name of the property", required=True)
    description = fields.Text(string="Property description")
    postcode = fields.Char(string="Postal code")
```

Adding string property is a good practice as it prove more info about the field